Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM fedora:27
+FROM fedora:28
 
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 ENV container docker
 
-# nettle update is necessary for dnsmasq which is used by libvirt
-RUN dnf -y update nettle && \
```

Comment:
Are we sure nettle is not needed anymore?

Reply:
Quite sure the command doesn't do anything nowadays.  It was probably trying to update from some buggy version.  Nettle is installed in the base image and if it is not at any point in the future, then it is packaging problem. That would be found pretty quickly.
Let me know if you want this hunk removed.

Reply:
Works for me for now.